qubesadmin/tests/mock_app.py

```diff
@@ -496,6 +581,37 @@ def __init__(self, qapp: QubesTest, dev_class: str,
         self.backend_vm = backend_vm
         self.attached = attached
 
+        self.update_calls()
+
+    def device_string(self):
+        if self.dev_class == 'block':
+            return f'{self.dev_id} mode=w device_node=/dev/{self.dev_id} ' \
```

Comment:
hm, I don't see `device_node` in the output from an actual system. Here are two examples for block:
```
sda1:0000:0000::?******:1 device_id='0000:0000::?******:1' port_id='sda1' devclass='block' backend_domain='dom0' serial='QEMU HARDDISK' manufacturer='sub-device of dom0:sda' interfaces='b******' parent_port_id='sda' parent_devclass='block'
```
```
loop0::0 device_id=':0' port_id='loop0' devclass='block' backend_domain='testvm' serial='/root/test.img' manufacturer='hosted by testvm' interfaces='b******'
```

The first one is a sub-device, if you want that. The second one is a plain device.